src/mapping/grouping/attractor_mapping_featurizing.jl

```diff
@@ -166,6 +166,12 @@ function extract_features_single(mapper, ics; show_progress = true, N = 1000)
     return feature_vector
 end
 
+# Mapper function that works with GroupViaNearestFeature
```

Comment:
This comment is missleading. This function also works with the histogram grouping. That's why I don't like comments above functions. We should remove this comment.